cms/api.py

```diff
@@ -168,6 +169,52 @@ def ensure_program_product_index() -> cms_models.ProgramIndexPage:
     return program_index
 
 
+def ensure_signatory_index() -> cms_models.ProgramIndexPage:
+    """
+    Same as ensure_product_index, but operates on programs instead.
+    """
+    home_page = get_home_page()
+    signatory_index = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(cms_models.SignatoryIndexPage)
+    ).first()
+    if not signatory_index:
+        signatory_index = cms_models.SignatoryIndexPage(title="Signatories")
+        home_page.add_child(instance=signatory_index)
+        signatory_index.save_revision().publish()
+
+    if signatory_index.get_children_count() != cms_models.SignatoryPage.objects.count():
+        for signatory_page in cms_models.SignatoryPage.objects.all():
+            signatory_page.move(signatory_index, "last-child")
+        log.info("Moved signatory pages under signatory index page")
+    return signatory_index
+
+
+def ensure_certificate_index() -> cms_models.ProgramIndexPage:
+    """
+    Same as ensure_product_index, but operates on programs instead.
+    """
+    home_page = get_home_page()
+    certificate_index = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(cms_models.CertificateIndexPage)
+    ).first()
```

Comment:
Can this be replaced with:

`    certificate_index = cms_models.CertificateIndexPage.objects.first()
`